src/chains/serializers.py

```diff
@@ -27,8 +28,13 @@ class GasPriceFixed1559Serializer(serializers.Serializer[GasPrice]):
     max_priority_fee_per_gas = serializers.CharField()
 
 
-class GasPriceSerializer(serializers.Serializer[GasPrice]):
-    def to_representation(self, instance: GasPrice) -> ReturnDict:
+class GasPriceSerializer(serializers.ModelSerializer[GasPrice]):
+    class Meta:
+        fields = "__all__"
```

Comment:
I think some of the fields of `GasPrice` should not be open for serialisation. (e.g. the `rank` can be seen as private field).

I'd suggest the following:

```python
fields = ["oracle_uri", "oracle_parameter", "gwei_factor", "fixed_wei_value", "max_fee_per_gas", "max_priority_fee_per_gas"]
```

Reply:
Fixed in 4b2a288431b8ec3ab414b4c7f2e7a99eedc8e547.